docs/source/scaleVM.md

```diff
@@ -0,0 +1,210 @@
+
+## 🧰 Fedora VM BootStorm (Ephemeral Storage)
+
+This mode loads Fedora VMs using ephemeral storage and verifies their readiness via the login page.
+
+### Fedora-Specific Parameters
+
+| Variable | Description                                                                                      |
+|----------|--------------------------------------------------------------------------------------------------|
+| `BOOTSTORM_SCALE` | Number of Fedora VMs per worker node, e.g. 10                                                    |
+| `FEDORA_CONTAINER_DISK` | (Optional) Fedora .qcow2 image from quay.io (e.g.: quay.io/openshift-cnv/qe-cnv-tests-fedora:37) |
+
+🔧 Example Configuration:
+```bash
+# OpenShift admin password
+export KUBEADMIN_PASSWORD=xxxxx-xxxxx-xxxxx-xxxxx
+# Number of Windows VMs per worker node
+export BOOTSTORM_SCALE=10
+# container disk image
+export FEDORA_CONTAINER_DISK="quay.io/openshift-cnv/qe-cnv-tests-fedora:37"
+# List of target worker nodes
+export SCALE_NODES="[ 'worker-0', 'worker-1', 'worker-2' ]"
+# Number of physical CPUs to use for parallelism
+export THREADS_LIMIT=20
+# Path to kubeconfig
+export KUBECONFIG_PATH=/root/.kube/config
+# Timeout in seconds (0 or negative means no timeout)
+export TIMEOUT=7200
+# Path for logs (YAML files and benchmark_runner.log)
+export LOGS_PATH=/tmp/benchmark-runner-run-artifacts
+```
+
+### Example Script for scaling Fedora VMs
+
+```bash
+#!/bin/bash
+
+# Load Fedora VMs including verification (verify login page)
+sudo podman run --rm -it \
+  -e WORKLOAD='bootstorm_vm' \
+  -e KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD}" \
+  -e RUN_ARTIFACTS_URL="${RUN_ARTIFACTS_URL}" \
+  -e SAVE_ARTIFACTS_LOCAL=True \
+  -e SCALE="${BOOTSTORM_SCALE}" \
+  -e FEDORA_CONTAINER_DISK="${FEDORA_CONTAINER_DISK}" \
+  -e SCALE_NODES="${SCALE_NODES}" \
+  -e THREADS_LIMIT="${THREADS_LIMIT}" \
+  -e RUN_STRATEGY=True \
+  -e TIMEOUT="${TIMEOUT}" \
+  -e log_level="INFO" \
+  -v "${LOGS_PATH}:${LOGS_PATH}" \
+  -v "${KUBECONFIG_PATH}:${KUBECONFIG_PATH}" \
+  quay.io/ebattat/benchmark-runner:latest
+```
+Verify the running VMs and get bootstorm_time(millisecond):
+```
+$ oc get vmi -n benchmark-runner
+$ grep bootstorm_time benchmark_runner.log
+{'vm_name': 'bootstorm-vm-cd2a4395-1', 'node': 'worker-2', 'bootstorm_time': 577.893, 'vm_ssh': 1}
+```
+
+## 🧰 Windows VM Bootstorm (ODF Storage)
+
+This mode loads Windows VMs using ODF persistent volumes and validates successful login.
+
+### Windows-Specific Parameters
+
+| Variable | Description                                                                                 |
+|----------|---------------------------------------------------------------------------------------------|
+| `WINDOWS_SCALE` | Number of Windows VMs per worker node, e.g. 10                                              |
+| `WINDOWS_URL` | URL to the Windows .qcow2 image shared via NGINX (e.g., `http://<host>:8083/windows.qcow2`) |
+
```

Comment:
Should you note there is essentially no default (we can't provide access to prebuilt Windows qcow2s)

Reply:
I have added more clear description:
`| `WINDOWS_URL` | *(No default, must be provided by the user)* URL to the Windows `.qcow2` image shared via NGINX (e.g., `http://<host>:8083/windows.qcow2`) |`